__tests__/version.js

```diff
@@ -0,0 +1,25 @@
+// Versioning schema: http://semver.org/
```

Comment:
Automate rather than enforce this. You could use [git pre-commit hooks](https://git-scm.com/book/en/v2/Customizing-Git-Git-Hooks) to control the version numbers in both files, and save yourself the redundancy and potential conflicts.